### PR TITLE
Field Stone Addition and accompanying audit of field stone walls

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary
None

#### Purpose of change
Adds field stones as a specific type of stone that can be found commonly in New England in both creek beds and in fieldstone walls.  Field stones are large and flat differing from boulders and rocks.

#### Describe the solution
Adds field stones, audits field stone walls, creates deconstructs for fieldstone walls. No skill requirements to deconstruct field stone walls.

#### Describe alternatives you've considered
Removing supports roof because while it is possible I'm not sure we have a good way to model the resource costs of this.

### CODE REVIEW

File: data/json/itemgroups/SUS/creek_bed.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[
+  {
+    "id": "creek_bed",
+    "type": "item_group",
+    "items": [ [ "rock", 5 ], [ "field_stone", 20 ], [ "stick", 10 ], [ "long_stick", 2 ], [ "log", 1 ] ]
+  }
+]
```
Comment (Maleclypse) [inline data/json/itemgroups/SUS/creek_bed.json:7]:
```suggestion
]

```

File: data/json/deconstruction.json [hunk 2]
```diff
@@ -462,6 +462,28 @@
     "pre_terrain": "t_rock_wall_half",
     "post_terrain": "t_dirt"
   },
+  {
+    "type": "construction",
+    "id": "constr_remove_dry_rock_wall_halfway",
+    "group": "remove_rock_wall",
+    "//": "Cuts out a segment of half complete stone wall, and then breaks it apart for disposal.",
+    "category": "DECONSTRUCT",
+    "time": "180 m",
```
Comment (Maleclypse) [inline data/json/deconstruction.json:471]:
```suggestion
    "time": "180 m",
    "required_skills": [ [ "fabrication", 0 ] ],
```

File: data/json/deconstruction.json [hunk 3]
```diff
@@ -462,6 +462,28 @@
...
+    "pre_terrain": "t_drystone_wall_half",
+    "post_terrain": "t_dirt"
+  },
+  {
+    "type": "construction",
+    "id": "constr_remove_dry_rock_wall",
+    "group": "remove_rock_wall",
+    "//": "Cuts out a segment of half complete stone wall, and then breaks it apart for disposal.",
+    "category": "DECONSTRUCT",
+    "time": "360 m",
```
Comment (Maleclypse) [inline data/json/deconstruction.json:482]:
```suggestion
    "time": "360 m",
    "required_skills": [ [ "fabrication", 0 ] ],
```

File: data/json/items/resources/stone.json [hunk 4]
```diff
@@ -29,6 +29,20 @@
     "name": { "str": "flint" },
     "description": "A chunk of flint, the kind of rock that can make sparks if stuck against high-carbon steel.  It also chips in a semi-predictable way if struck, leaving a sharp edge."
   },
+  {
+    "type": "GENERIC",
+    "id": "field_stone",
+    "symbol": ";",
+    "color": "brown",
+    "name": { "str": "field stone" },
+    "description": "A medium, flatish, heavy rock, big enough and flat enough to be used as a into field stone walls.",
```
Comment (Vaskritaya) [inline data/json/items/resources/stone.json:38]: seems like youre missing a bit in the end of the description. used as a what?

Comment (Maleclypse) [reply data/json/items/resources/stone.json:38]: Thanks! I’ll need to rewrite that

Comment (Maleclypse) [inline data/json/items/resources/stone.json:38]:
```suggestion
    "description": "A medium, flatish, heavy rock, big enough and flat enough to be used as a block in field stone walls.",
```

File: data/json/itemgroups/SUS/creek_bed.json [hunk 5]
```diff
@@ -0,0 +1,7 @@
+[
+  {
+    "id": "creek_bed",
+    "type": "item_group",
+    "items": [ [ "rock", 5 ], [ "field_stone", 20 ], [ "stick", 10 ], [ "long_stick", 2 ], [ "log", 1 ] ]
```
Comment (Maleclypse) [inline data/json/itemgroups/SUS/creek_bed.json:5]:
```suggestion
    "items": [ [ "rock", 5 ], [ "field_stone", 20 ], [ "stick", 10 ], [ "stick_long", 2 ], [ "log", 1 ] ]
```

**Pull request merged**